java/code/src/com/redhat/rhn/frontend/xmlrpc/system/config/ServerConfigHandler.java

```diff
@@ -72,7 +69,6 @@
  */
 public class ServerConfigHandler extends BaseHandler {
 
-    private static Logger log = LogManager.getLogger(ServerConfigHandler.class);
```

Comment:
I know the reasoning behind removing this as it is not used, but for some, let's say, ease of development shouldn't we instead of removing these lines add some usage so when someone new will need logging they will not need to look how to use the logger?

Or maybe add some code bits to wiki so it can be reused.
Given how many files you needed to change to standardize on logging, having already present object or documentation is beneficial.

Reply:
That is an option indeed... Not sure I'll have time to add them all soon though